Xamarin-iOS/ProgressBar/GettingStarted.md

```diff
@@ -15,7 +15,7 @@ This section explains the steps required to work with progress bar in Xamarin.iO
 ## Add progress bar reference
 
 After installing Essential Studio® for Xamarin, you can find all the required assemblies in the installation folders, typically:
-{Syncfusion Installed location}\Essential Studio{version number}\lib
+{Syncfusion&reg; Installed location}\Essential Studio&reg;{version number}\lib
```

Comment:
Whether the trade mark is need in the Essential Studio in this place?

Reply:
I have removed